index.html

```diff
@@ -338,6 +365,7 @@
 if (localStorage.history && localStorage.history.length > 0) {
   box.innerHTML = localStorage.history;
   box.scrollTop = box.scrollHeight - 10;
+  loadRoster(box);
```

Comment:
Don't do this, it's too good, log the speakers to `localStorage.roster`, e.g., `localStorage.roster.push(obj.nick)`